README.md

```diff
@@ -25,6 +25,7 @@ align="right">
 * [Using the queue module](#using-the-queue-module)
   * [Initialization](#initialization)
   * [Creating a new queue](#creating-a-new-queue)
+  * [Set queue settings](#set-queue-settings)
```

Comment:
We usually mark relevant commits that follows `Closes #xxx` as `Follows up #xxx` rather than `Part of #xxx`.

Reply:
Done.